.env

```
@@ -0,0 +1,40 @@
+PGHOST=plane-db
+PGDATABASE=plane
+POSTGRES_USER=colton84
+POSTGRES_PASSWORD=hk3nhmqkydastxnkezqdelfwefzylxvm
+POSTGRES_DB=plane
+POSTGRES_PORT=5432
+PGDATA=/var/lib/postgresql/data
+REDIS_HOST=plane-redis
+REDIS_PORT=6379
+REDIS_URL=redis://plane-redis:6379/
+MINIO_ROOT_USER=access-key
+MINIO_ROOT_PASSWORD=e7rld1rryvhtjzephnxbzwhycurburji
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The MINIO_ROOT_PASSWORD key should go before the MINIO_ROOT_USER key

(UnorderedKey)